meson_options.txt

```diff
@@ -0,0 +1,188 @@
+## SPDX-License-Identifier: 0BSD
+## Author: Vincent Torri
```

Comment:
Makefile.am files use `##` because Automake omits such lines from the produced Makefile.in files; lines with a single `#` get into Makefile.in. So there is no need to use `##` in meson.build unless it was a stylistic preference.

Reply:
i'll change that if you want. I just copy/pasted this from muon and there was 2 #

Reply:
fixed in efcb02e6fa110cbadac1b1a84fd2ba3674983142